pong_game/ball.tscn

```diff
@@ -31,11 +29,3 @@ shape = SubResource("CircleShape2D_sntrn")
 [node name="Sprite2D" type="Sprite2D" parent="."]
 unique_name_in_owner = true
 texture = ExtResource("2_xkrmm")
-
-[node name="WallAudioStreamPlayer" type="AudioStreamPlayer" parent="."]
-stream = ExtResource("3_mjbsd")
-
-[node name="PaddleAudioStreamPlayer" type="AudioStreamPlayer" parent="."]
-stream = ExtResource("4_skr8k")
-
-[connection signal="body_entered" from="." to="." method="_on_body_entered"]
```

Comment:
Where is the orphan method defined? Should that also be removed?

Reply:
The method does not exist, so the signal binding is an orphan